source/features/profile-gists-link.css

```diff
@@ -1,5 +1,4 @@
 /* Fix wrapping text on tabs when feature is enabled #1757 */
-.UnderlineNav-item:nth-last-child(n+7),
-.UnderlineNav-item:nth-last-child(n+7) ~ * {
+.UnderlineNav-item:is(:nth-last-child(n+7), :nth-last-child(n+7) ~ *) {
```

Comment:
@sindresorhus thoyghts on the readability of :is() selectors? 

Some are good, some aren’t. Probably it just depends on how simple the selectors inside :is() are. In this case I’d give a 👎